materials/urls.py

```diff
@@ -0,0 +1,20 @@
+from rest_framework.routers import SimpleRouter
+from django.urls import path
+
+from materials.views import CourseViewSet, LessonListAPIView, LessonUpdateAPIView, LessonRetrieveAPIView, LessonCreateAPIView, LessonDestroyAPIView
+from materials.apps import MaterialsConfig
+
+app_name = MaterialsConfig.name
+
+router= SimpleRouter()
+router.register(prefix='', viewset=CourseViewSet, basename='course')
+
+urlpatterns = [
+    path("lesson/", LessonListAPIView.as_view(),name="lesson_list") ,
+    path("lesson/<int:pk>/", LessonRetrieveAPIView.as_view(),name="lesson_retrive"),
+    path("lesson/create", LessonCreateAPIView.as_view(), name="lesson_create"),
```

Comment:
У всех эндпоинтов должен быть слеш в конце
```suggestion
    path("lesson/create/", LessonCreateAPIView.as_view(), name="lesson_create"),
```